specification/azsadmin/resource-manager/infrastructureinsights/Microsoft.InfrastructureInsights.Admin/stable/2016-05-01/InfrastructureInsights.json

```diff
@@ -0,0 +1,336 @@
+{
+    "swagger": "2.0",
```

Comment:
>     [](start = 0, length = 4)

Could 2 space idents be used instead?

Reply:
Is there a technical reason that 2 spaces is preferred?  I prefer 4 spaces.

Reply:
@deathly809 @mcardosos  I would recommend to have a separate PR to fix an indent after this one. Please, leave the indent as it is for this PR so it's easy to review.

Reply:
I would prefer the 2 spaces indent to be fixed here, actually.

Reply:
I prefer 4 spaces and there is no requirement on spacing.

Reply:
Personally, in general I prefer 4 spaces too. But consistency on the repo is important, and it makes reviewing a lot easier / less noisy / faster.

Reply:
So far my PRs have been 4 spaces.  No one has been able to point to a requirement and I have a hard time reading 2 spaces.